graphein/protein/graphs.py

```diff
@@ -493,6 +493,13 @@ def initialise_graph_with_metadata(
             sequence = protein_df.loc[protein_df["chain_id"] == c][
                 "residue_name"
             ].str.cat()
+        elif granularity == "atom":
+            protein_df = protein_df.loc[protein_df["atom_name"] == "CA"]
```

Comment:
Hmm, is keeping the CA subset of `protein_df` not problematic downstream?

Do we not want to store this in a temporary variable (if at all, can we not just add the atom_name selection into L499 when we select by chain ID?

Reply:
Yep, 100%.  My bad, I forgot that when it's passed to the graph construction it only passes a reference to it.